torch/_inductor/codecache.py

```diff
@@ -1987,10 +1987,15 @@ def _compile_consts_linux(consts: bytes) -> str:
                 # as read-only (i.e. .lrodata) which could accomodate larger size of data
                 # to be linked.
                 rename_data = " .data=.lrodata,alloc,load,readonly,data,contents"
+
+            from .codegen.memory_planning import ALIGN_BYTES
+
+            # following the gAlignment of CPU in c10/core/alignment.h
+            assert ALIGN_BYTES == 64, "Expect ALIGN_BYTES to be 64 for CPU"
```

Comment:
In `memory_planning.py`, the `ALIGN_BYTES` is already set to 64, which is what we expect. If better to add a separate alignment value for CPU, I could add another variable. If the current way is fine, I'll keep it as is.
https://github.com/pytorch/pytorch/blob/f4d7cdc5e63c786b1f6588eafa53bbc6d33c3826/torch/_inductor/codegen/memory_planning.py#L25

Reply:
What `ALIGN_BYTES == 64` be too strict? Is `ALIGN_BYTES > 1 and ALIGN_BYTES % 64 == 0` better?

Reply:
Also, perhaps it is better to move the ALIGN_BYTES to a more common place?